src/router/pam/automations.js

```diff
@@ -85,10 +85,10 @@ export default [
       {
         path: 'account/:id',
         component: () => import('@/views/accounts/AccountDiscover/AccountDetail/index.vue'),
-        name: 'AccountDiscoverAccountDetail',
+        name: 'DiscoverAccountDetail',
         hidden: true,
         meta: {
-          title: i18n.t('AccountDiscoverAccountDetail'),
+          title: i18n.t('DiscoverAccountDetail'),
           permissions: ['accounts.view_gatheraccountsexecution']
         }
       }
```

Comment:
The only difference between the original version and current version is that all of them use "i18n" instead of "lgbm", hence there isn't really an issue but just a typo, not necessarily irregularity.

Optimization Suggestions:

As no additional requirements were provided for optimization, the suggestion would be to review if any changes are needed, and perhaps refactor the code structure (if necessary) keeping readability and efficiency in mind.